docs/user-manual/modules/ROOT/pages/camel-jbang-kubernetes.adoc

```diff
@@ -180,6 +180,31 @@ The Kubernetes plugin export command also inherits all options from the arbitrar
 
 TIP: See the possible options by running: `camel kubernetes export --help` for more details.
 
+=== Kubernetes Container Health Probes
+
+The https://camel.apache.org/components/next/others/observability-services.html[Observability Services] component is set by default in the project created by the export command, then the health path probes are prefixed with the `/observe` endpoint, regardless if they are Quarkus or Spring Boot.
+
+These are the HTTP Container probe endpoints.
+
+For Quarkus.
+[source]
+----
+/observe/health/live
+/observe/health/ready
+/observe/health/started
+----
+
+For Spring Boot (there is no startup probe)
+[source]
+----
+/observe/health/liveness
```

Comment:
These should be the same of the ones we have for the other runtimes. It is possible that the springboot also exposes those as an alias, but we should make use of the harmonized ones.